Assets/Scripts/Utils/DrawArrow.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Draws debug arrows for vectors.
+/// </summary>
+public static class DrawArrow
```

Comment:
don't name a class as you would a method (with a verb)

name it "GizmoArrow" , as a noun